hello-data-subsystems/hello-data-cloudbeaver-gateway/src/main/resources/application.yml

```diff
@@ -63,10 +63,10 @@ spring:
         url: http://localhost:9090
         username: "user"       #These two are needed so that the client
         password: "password"   #can register at the protected server api
-        instance:
-          metadata:
-            user.name: "user"         #These two are needed so that the server
-            user.password: "password" #can access the protected client endpoints
+  #        instance:
```

Comment:
Maybe we can remove the commented out code

Reply:
Wanted to leave that as an example of what we need to send to SBA so that it can log in to the app. What do you think?